### PR TITLE
Typo and docstring

### PR DESCRIPTION
**Pull request by adamrp**
minor, noticed while reviewing other PRs

### CODE REVIEW

File: qiita_db/metadata_template.py [hunk 1]
```diff
@@ -1181,9 +1181,8 @@ def create(cls, md_template, study):
         ----------
         md_template : DataFrame
             The metadata template file contents indexed by samples Ids
-        study : Study or RawData
-            The study to which the metadata template belongs to. Study in case
-            of SampleTemplate and RawData in case of PrepTemplate
+        study : Study
+            The study to which the sample template belongs to.
```
Comment (adamrp) [inline qiita_db/metadata_template.py]: I think this is an accurate change, but please correct me if I'm wrong, and I can revert

Comment (josenavas) [inline qiita_db/metadata_template.py]: Yes it is accurate. Good catch!

File: qiita_pet/handlers/study_handlers/description_handlers.py [hunk 2]
```diff
@@ -77,7 +77,7 @@ def _to_int(value):
 
 class StudyDescriptionHandler(BaseHandler):
 
-    def _get_sudy_and_check_access(self, study_id):
+    def get_study_and_check_access(self, study_id):
```
Comment (josenavas) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: Why is no longer private?

Comment (adamrp) [inline qiita_pet/handlers/study_handlers/description_handlers.py]: Woops, find/replace mistake. correcting now...

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**